python/ccf/proposal_generator.py

```diff
@@ -171,11 +171,13 @@ def build_proposal(
             vote_lines.append("  let args = action.args;")
 
             for name, body in args.items():
+                vote_lines.append("{")
                 vote_lines.append(f"  if (!('{name}' in args)) {{ return false; }};")
-                vote_lines.append(f"  var expected = {json.dumps(body)};")
+                vote_lines.append(f"  let expected = {json.dumps(body)};")
                 vote_lines.append(
                     f"  if (JSON.stringify(args['{name}']) !== JSON.stringify(expected)) {{ return false; }};"
                 )
+                vote_lines.append("}")
```

Comment:
There's an attempt at indentation in the produced scripts - we should keep that here. It won't let me suggest and edit, but:

```
                vote_lines.append("  {")
                vote_lines.append(f"    if (!('{name}' in args)) {{ return false; }};")
                vote_lines.append(f"    let expected = {json.dumps(body)};")
                vote_lines.append(
                    f"    if (JSON.stringify(args['{name}']) !== JSON.stringify(expected)) {{ return false; }};"
                )
                vote_lines.append("  }")
```